org.civicrm.reqangular/src/common/services/notification.js

```diff
@@ -0,0 +1,56 @@
+(function (CRM) {
+  define([
+    'common/modules/services'
+  ], function (services) {
+    'use strict';
+
+    services.factory('notification', function () {
+      return {
+
+        /**
+         * Uses CRM.alert to display alert notification
+         *
+         * @param {string} title
+         * @param {string} body
+         * @param {object} options
+         */
+        alert: function (title, body, options) {
```

Comment:
To me it looks like a lot of repetition in the file. Would it be better to do something like this instead?

```js
(function (CRM) {
  define([
    'common/modules/services'
  ], function (services) {
    'use strict';

    services.factory('notification', function () {
      var notificationFunctions = {};
      var notificationTypes = ['alert', 'success', 'info', 'error'];

      notificationTypes.forEach(function (notificationType) {
        notificationFunctions[notificationType] = function (title, body, options) {
          return CRM.alert(body, title, notificationType, options);
        }
      });

      return notificationFunctions;
    })
  });
}(CRM));
```

With lodash it could be done even more elegant, but I am not sure we want to depend on lodash just for this.